engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -373,6 +373,18 @@ paths:
           required: true
           type: string
           in: path
+        - name: outputs
+          description: Include output metadata
+          required: false
+          type: boolean
+          in: query
+          default: true
+        - name: timings
```

Comment:
Too late for this PR, as this PR is a hotfix/copypasta of #761.

Overall, this was implemented based on comments from #719, but a new ticket that outlines expected behavior could easily be done.